pkg/controller/expansion/expansion_controller.go

```diff
@@ -91,23 +104,36 @@ func newReconciler(mgr manager.Manager, system *expansion.System, getPod func(ct
 		statusClient: mgr.GetClient(),
 		getPod:       getPod,
 		tracker:      tracker,
+		events:       ev,
+		eventSource:  &source.Channel{Source: ev, DestBufferSize: 1024},
 	}
 }
 
-func add(mgr manager.Manager, r reconcile.Reconciler) error {
+func add(mgr manager.Manager, r *Reconciler) error {
 	c, err := controller.New("expansion-template-controller", mgr, controller.Options{Reconciler: r})
 	if err != nil {
 		return err
 	}
 
+	// Watch for enqueued events
+	if r.eventSource != nil {
+		err = c.Watch(
+			r.eventSource,
+			&handler.EnqueueRequestForObject{})
+		if err != nil {
+			return err
+		}
+	}
+
+	// Watch for changes to ExpansionTemplates
 	return c.Watch(
 		&source.Kind{Type: &v1alpha1.ExpansionTemplate{}},
 		&handler.EnqueueRequestForObject{})
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	defer r.registry.report(ctx)
-	log.Info("Reconcile", "request", request, "namespace", request.Namespace, "name", request.Name)
+	log.V(logging.DebugLevel).Info("Reconcile", "request", request, "namespace", request.Namespace, "name", request.Name)
```

Comment:
do we want to do this for other reconcile logs too?

Reply:
The other reconcilers only log if they encounter an error or an otherwise "interesting" situation. Since expansion is in alpha, I thought it would be useful to log every request here, which would help us debug if an issue comes up. I think we eventually remove this log line once we graduate the feature to beta or v1, as it is quite chatty.